server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/data/entities/WUIRiskClassCodeEntity.java

```diff
@@ -23,7 +25,7 @@
 import lombok.NoArgsConstructor;
 
 @Entity
-@Table(name = "wui_risk_class_code")
+@Table(name = "wui_risk_class_rank")
```

Comment:
Why is this being changed? I don't think this is correct. You should adding a new entity/model/resourceassembler etc for WUIRiskClassRank. WUIRiskClassCode is for the code table?

Reply:
I looked in to this table and find we are not using this wui_risk_class_code table anywhere in the app, it does not has weighted value and wui_risk_class_rank is actually what we need.